flink-runtime/src/main/java/org/apache/flink/runtime/leaderelection/DefaultLeaderElection.java

```diff
@@ -56,4 +56,40 @@ public boolean hasLeadership(UUID leaderSessionId) {
     public void close() throws Exception {
         parentService.remove(contenderID);
     }
+
+    /**
+     * {@link ParentService} is introduced to define the protocol between {@link
```

Comment:
```suggestion
     * {@link ParentService} is defines the protocol between {@link
```
nit: I guess we should describe the current state in JavaDoc rather than how we got there

Reply:
Done.